<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n de Directores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../dire.png</normaloff>../dire.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 127);
background-color: rgb(0, 85, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>229</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>5</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">;
background-color: rgb(255, 252, 162);</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>157</y>
      <width>521</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>4</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="btnGrabar">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>14</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">;
background-color: rgb(255, 208, 181);</string>
     </property>
     <property name="text">
      <string>Grabar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancelar">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>14</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>368</x>
       <y>14</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNuevo">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>14</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>Nuevo</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnWikipedia">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>88</y>
      <width>51</width>
      <height>43</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string notr="true">Ir a Wikipedia</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Wikipedia.png</normaloff>../Wikipedia.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>53</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter Display</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
    <property name="indent">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>58</y>
      <width>429</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="nombreCineasta">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter Display Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wikiDirector">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter Display Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string extracomment="Sitio"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>18</y>
      <width>137</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>5</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter Display SemiBold</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelId_Director">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>5</width>
         <height>11</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter Display SemiBold</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>btnWikipedia</zorder>
   <zorder>label_2</zorder>
  </widget>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo </string>
   </property>
  </action>
  <action name="actionModificar">
   <property name="text">
    <string>Modificar</string>
   </property>
  </action>
  <action name="actionCancelar">
   <property name="text">
    <string>Cancelar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionModificar_2">
   <property name="text">
    <string>Modificar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
